src/skia/Font.cpp

```diff
@@ -65,6 +67,12 @@ void SetVariationPositionCoordinates(
     vp.coordinateCount = coords.size();
 }
 
+void SetPaletteOverrides(
+    SkFontArguments::Palette& palette,
+    const std::vector<Override>& overrides) {
+    palette.overrides = &overrides[0];
```

Comment:
This can segfault intermittently as it is valid to pass in `std::vector<Override>()` (a zero length `std::vector`).

Reply:
This is basically a copy of the `SetVariationPositionCoordinates()` which does not do any size checks either.

Reply:
There are two issues here: does skia itself segfaults with zero length input? And 2nd issue, pybind11 definitely allows you to pass a zero length array into this code, and it would segfault in python before it gets to skia.

Reply:
The infuriating thing is that this would intermittently segfault, depends on your python interpreter's memory layout at the moment.

Reply:
I looked up the c++ standard recently because of this basically `&overrides[0]` can point at any thing within python/pybind11, and can segfault intermittently, depending on your current memory layout.